.github/ISSUE_TEMPLATE/exoit-magic-leap-bug-report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Exoit Magic Leap Bug report
+about: File an issue to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
Maybe more practically phrased as “what happened, and what did you expect to happen”?